docs/components/ComponentPropsTable.tsx

```diff
@@ -43,6 +43,27 @@ export const ComponentPropsTable = ({ data }: ComponentPropsTableProps) => (
 		</TableHead>
 		<TableBody>
 			{Object.values(data.props).map((prop) => {
+				let defaultValue = prop.defaultValue?.value;
+
+				if (defaultValue) {
+					if (
+						prop.type.name === 'boolean' ||
+						(typeof defaultValue === 'string' && defaultValue.startsWith('['))
+					) {
+						// Do nothing
+					} else if (
```

Comment:
This fixes the default `value` prop for the Time input https://design-system.agriculture.gov.au/pr-preview/pr-1582/components/time-input/code